plugins/chruby/chruby.plugin.zsh

```diff
@@ -95,5 +95,11 @@ function chruby_prompt_info() {
 }
 
 # complete on installed rubies
-_chruby() { compadd $(chruby | tr -d '* ') }
+_chruby() {
+    compadd $(chruby | tr -d '* ')
+    local default_path='/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin'
+    if ( PATH=${default_path} type ruby > /dev/null); then
```

Comment:
This should be `&>/dev/null` to redirect also stderr. You can also get rid of the parentheses.